Seemingly can't login

Hi! At some point it started outputting this:
Can't get cups: token not found. Cannot continue
Looks like they changed the webpage or am I the only one?
Thanks.

No luck :(

I have modified the function like this:

```
    def __get_token(self):
        r = self.__get_url('https://zonaprivada.edistribucion.com/areaprivada/s/')
        soup = BeautifulSoup(r.text, 'html.parser')
        # logging.info(soup)
        scripts = soup.find_all('script')
        logging.info(f'Loading token scripts ({len(scripts)})')
        for s in scripts:
            if 'auraConfig' in str(s):
                logging.info("Found 'auraConfig'")
                logging.info(f"s =\n{str(s)[:300]}")
                logging.info(f"s.text =\n{s.text}")
            if True: #if ('auraConfig' in s.text):
                prsr = jsparse(s.text)
                for b in prsr['body']:
                    decls = b.get('expression', {}).get('callee', {}).get('body', {}).get('body', [])
                    for d in decls:
                        if (d.get('type', None) == 'VariableDeclaration'):
                            for dc in d.get('declarations', []):
                                if (dc.get('id', {}).get('name', None) == 'auraConfig'):
                                    for prop in dc.get('init', {}).get('properties', []):
                                        if (prop.get('key', {}).get('value', None) == 'token'):
                                            return prop.get('value', {}).get('value', None)
        logging.error("Could not find token!")
        return None
```

And the relevant log that this outputs is:

```
[21/07/2023 19:02:16] [INFO] Sending GET request to https://zonaprivada.edistribucion.com/areaprivada/s/
[21/07/2023 19:02:16] [INFO] Response with code: 200
[21/07/2023 19:02:16] [INFO] Loading token scripts (11)
[21/07/2023 19:02:16] [INFO] Found 'auraConfig'
[21/07/2023 19:02:16] [INFO] s =
<script nonce="yNmRuC3HJ1L45uF8FmxeYaRnlSVLYX7o">
(function () {
window.pageStartTime = (new Date()).getTime();
window.Aura || (window.Aura = {});
window.Aura.bootstrap || (window.Aura.bootstrap = {});


var time = window.performance && window.performance.now ? window.performance.now.bind(performanc
[21/07/2023 19:02:16] [INFO] s.text =

[21/07/2023 19:02:16] [ERROR] Could not find token!
Traceback (most recent call last):
  File "/home/pi/edistribucion/./meter.py", line 29, in <module>
    edis = Edistribucion(login, password, logging.INFO)
  File "/home/pi/edistribucion/src/EdistribucionAPI/Edistribucion.py", line 133, in __init__
    self.login()
  File "/home/pi/edistribucion/src/EdistribucionAPI/Edistribucion.py", line 240, in login
    return self.__force_login()
  File "/home/pi/edistribucion/src/EdistribucionAPI/Edistribucion.py", line 320, in __force_login
    raise EdisError('token not found. Cannot continue')
EdistribucionAPI.Edistribucion.EdisError: token not found. Cannot continue
```

--- a/src/EdistribucionAPI/Edistribucion.py
+++ b/src/EdistribucionAPI/Edistribucion.py
@@ -246,8 +246,8 @@ class Edistribucion():
         scripts = soup.find_all('script')
         logging.info('Loading token scripts')
         for s in scripts:
-            if ('auraConfig' in s.text):
-                prsr = jsparse(s.text)
+            if ('auraConfig' in s.string):
+                prsr = jsparse(s.string)
                 for b in prsr['body']:
                     decls = b.get('expression', {}).get('callee', {}).get('body', {}).get('body', [])
                     for d in decls:

